### PR TITLE
[R1065] initial implementation of skos:altLabel and skos:prefLabel on term detail

### PR DESCRIPTION
**Pull request by psiotwo**
https://kbss.felk.cvut.cz/redmine/issues/1065

Included:
- altLabels as a part of the header of term detail screen
- altLabels + hiddenLabels editable in term detail edit
- hiddenLabels not part of the term detail view

Not Included:
- fulltext search results

### CODE REVIEW

File: src/component/term/TermDetail.tsx [hunk 1]
```diff
@@ -93,7 +93,7 @@ export class TermDetail extends EditableComponent<TermDetailProps> {
         const buttons = this.getButtons();
         return <div id="term-detail">
             <HeaderWithActions title={
-                <>{term.label}<CopyIriIcon url={term.iri as string}/></>
+                <>{term.label}<CopyIriIcon url={term.iri as string}/><br/><h6>{Utils.sanitizeArray(term.altLabels || []).join(", ")}</h6></>
```
Comment (ledsoft) [inline src/component/term/TermDetail.tsx]: Calling `Utils.sanitizeArray(term.altLabels || [])` is superfluous, sanitization ensures that an array is returned, so there's no need for the conditional argument.

File: src/component/term/TermMetadataEdit.tsx [hunk 2]
```diff
@@ -99,6 +108,20 @@ export class TermMetadataEdit extends React.Component<TermMetadataEditProps, Ter
                                          help={i18n("term.label.help")}/>
                         </Col>
                     </Row>
+                    <Row>
+                        <Col xs={12}>
+                            <StringListEdit list={this.state.altLabels || []}
```
Comment (ledsoft) [inline src/component/term/TermMetadataEdit.tsx]: Why is the array sanitized here, when it is also sanitized in the `StringListEdit` component? This way, the list is IMHO sanitized unnecessarily multiple times. I suggest leaving the sanitization in the `StringListEdit` component, and not doing it here. Also, we should probably stick to a single method of sanitization - either the `value || []` or `Utils.sanitizeArray` (the second version being able to handle accidental singular value as well).

File: src/component/term/TermMetadataEdit.tsx [hunk 3]
```diff
@@ -99,6 +108,20 @@ export class TermMetadataEdit extends React.Component<TermMetadataEditProps, Ter
...
+                                            onChange={this.onAltLabelsChange}
+                                            i18nPrefix={"term.metadata.altLabels"}/>
+                        </Col>
+                    </Row>
+                    <Row>
+                        <Col xs={12}>
+                            <StringListEdit list={this.state.hiddenLabels || []}
```
Comment (ledsoft) [inline src/component/term/TermMetadataEdit.tsx]: DTTO

File: src/component/term/TermDetail.tsx (hunk 1 above)
Comment (psiotwo) [reply src/component/term/TermDetail.tsx]: fixed

File: src/component/term/TermMetadataEdit.tsx (hunk 2 above)
Comment (psiotwo) [reply src/component/term/TermMetadataEdit.tsx]: fixed

File: src/component/term/TermMetadataEdit.tsx (hunk 3 above)
Comment (psiotwo) [reply src/component/term/TermMetadataEdit.tsx]: fixed

**Review by ledsoft — APPROVED** [review]
LGTM

**Pull request merged**